scripts/vite/plugins/blsBrowserPlugin.ts

```diff
@@ -0,0 +1,32 @@
+import path from "node:path";
+import {UserConfig, ConfigEnv, Plugin} from "vite";
+const __dirname = new URL(".", import.meta.url).pathname;
+const polyfillsDir = path.join(__dirname, "../polyfills");
+
+const emptyModulePath = path.join(__dirname, "../polyfills/emptyModule.js");
+
+export function blsBrowserPlugin(): Plugin {
+  return {
+    name: "blsBrowserPlugin",
+    config(_config: UserConfig, _env: ConfigEnv) {
+      return {
+        optimizeDeps: {
+          exclude: ["@chainsafe/bls-keygen"],
+          force: true,
+        },
+        resolve: {
+          alias: {
+            "@chainsafe/bls/types": "@chainsafe/bls/types",
```

Comment:
is this line needed?

Reply:
Yes we need it otherwise `@chainsafe/bls/types` map to `@chainsafe/bls/herumi/types` which is not available.